game/state/city/vehiclemission.cpp

```diff
@@ -132,14 +120,20 @@ bool FlyingVehicleTileHelper::canEnterTile(Tile *from, Tile *to, bool, bool &, f
 	// Allow pathing into tiles with crashes
 	bool foundCrash = false;
 	bool foundScenery = false;
+	auto self = v.shared_from_this();
```

Comment:
'self' can be const?

Reply:
Done, although I'm not a fan of making every unchanging local non-reference variable a const.